UKHomeOffice/cerberus-service
# COP-4093: Add Panel component

## Description

Add a Panel component.

## To Test
1. Run `npm i`
2. Go to the "Issue target form"
3. Submit the form
3. You should see a green panel on the confirmation page


## Developer Checklist

\* Required
